apiserver/testing/fakecharmstore.go

```diff
@@ -83,3 +85,17 @@ func (s *CharmStoreSuite) UploadCharm(c *gc.C, url, name string) (*charm.URL, ch
 func (s *CharmStoreSuite) UploadCharmMultiSeries(c *gc.C, url, name string) (*charm.URL, charm.Charm) {
 	return testcharms.UploadCharmMultiSeries(c, s.Client, url, name)
 }
+
+type v1ToV2Locator struct {
+	v1Loc bakery.PublicKeyLocator
+}
+
+func (v2Loc v1ToV2Locator) PublicKeyForLocation(loc string) (*bakeryV2.PublicKey, error) {
```

Comment:
This seems like a simple wrapper - we can't modify our stuff to use v2?

Reply:
I've confirmed that we can remove bakery.v1 entirely and just use bakery.v2-unstable